src/emoji/__tests__/data-test.js

```diff
@@ -1,10 +1,6 @@
 /* @flow strict-local */
 import { codeToEmojiMap, getFilteredEmojis } from '../data';
 
-// Prettier disabled in .prettierignore ; it misparses this file, apparently
-// because of the emoji.  (Even if they're tucked away in comments, it still
-// gets it wrong.)
-
 /* eslint-disable no-multi-spaces */
```

Comment:
```
emoji tests [nfc]: Re-enable Prettier formatting

Whatever Prettier bug this was about, it seems to have been fixed.
```

Nice! Can we also remove this eslint-disable line now?

Reply:
Hmm, yeah. Turns out we can -- added a commit on top about that.